go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	github.com/mholt/archiver/v3 v3.5.1
 	github.com/moby/moby v27.5.1+incompatible
 	github.com/opencontainers/image-spec v1.1.1
+	github.com/phayes/freeport v0.0.0-20220201140144-74d24b5ae9f5
```

Comment:
I discovered that there is actually already a `helpers.GetAvailablePort()` function already, we can save on a dependency with this, looks to be essentially a copy of freeport.

Reply:
Good call!